### ISSUE TITLE
[iceprod2] global queue and distributed storage (Trac #1581)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1573]
Take the storage location for input files into account when queueing tasks, possibly through an extra priority factor?

Depends on #1572.

Migrated from https://code.icecube.wisc.edu/ticket/1581
```json
{
    "status": "closed", 
    "changetime": "2016-05-09T21:56:14", 
    "description": "Take the storage location for input files into account when queueing tasks, possibly through an extra priority factor?\n\nDepends on #1580.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "wontfix", 
    "_ts": "1462830974780090", 
    "component": "iceprod", 
    "summary": "[iceprod2] global queue and distributed storage", 
    "priority": "major", 
    "keywords": "", 
    "time": "2016-03-07T22:16:26", 
    "milestone": "Long-Term Future", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160330T14:55:01: 
* olivas changed status from "new" to "assigned"
* olivas commented:

> David automatically gets all IceProd 2 tickets, but should feel free to reassign to Delventhal.


* olivas changed owner from "" to "david.schultz"

**Comment by icecube-trac** [comment]
Trac update at 20160509T21:38:10: 
* david.schultz commented:

> moved to https://github.com/WIPACrepo/iceprod/issues


* david.schultz changed status from "assigned" to "closed"
* david.schultz changed resolution from "" to "wontfix"

**Comment by icecube-trac** [comment]
Trac update at 20160509T21:56:14: 
* david.schultz changed milestone from "IceProd 2.2" to "Long-Term Future"
* david.schultz commented:

> Milestone IceProd 2.2 deleted
